swift/Samples/BackStackContainer/Sources/BackStackContainer.swift

```diff
@@ -36,26 +35,26 @@ public final class BackStackContainer: ScreenViewController<BackStackScreen>, UI
         navController.didMove(toParent: self)
     }
 
-    public override func viewDidLayoutSubviews() {
+    override public func viewDidLayoutSubviews() {
         super.viewDidLayoutSubviews()
 
         navController.view.frame = view.bounds
     }
 
-    public override func screenDidChange(from previousScreen: BackStackScreen, previousEnvironment: ViewEnvironment) {
+    override public func screenDidChange(from _: BackStackScreen, previousEnvironment _: ViewEnvironment) {
```

Comment:
I vote for disabling the removal of unused params like this.

Reply:
+1

Reply:
disabling this rule.